index.html

```diff
@@ -96,12 +96,12 @@ <h2 class="tag">Software Developer</h2>
         </div>
```

Comment:
Please add a link to the corresponding line of code in the Pull Request changes If any issues are found.